templates/web/base/waste/bulky/payment_successful.html

```diff
@@ -0,0 +1,25 @@
+<!-- Step 9.2A Payment successful -->
```

Comment:
I tried using this template instead of the existing 'waste/bulky/confirmation.html', but it showed up with no styling for some reason 🤔 

<img width="340" alt="Screenshot 2022-10-19 at 18 39 49" src="https://user-images.githubusercontent.com/11098355/196765153-a4e6bd8b-65de-478c-abdd-e0f9f0834b0b.png">

Reply:
Thanks @nephila-nacrea I have added fixed both issues.
The styling might be because I wasn't sure where to include the templates. So for coding and visualisation purposes I include them on the bin_days.html template. But when I did the commits I got rid of that.

Reply:
I agree, these are just templates and still need all the logic to be included. I didn't put a warning because merging with master wouldn't have cause an issue. I have included a warning at the beginning about the backend logic being needed.